test/controllers/tcsp/OnlyOffTheShelfCompsSoldControllerSpec.scala

```diff
@@ -16,32 +16,31 @@
 
 package controllers.tcsp
 
-import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.mock.MockitoSugar
-import play.api.test.Helpers.{BAD_REQUEST, OK, SEE_OTHER, contentAsString, redirectLocation, status}
 import connectors.DataCacheConnector
+import controllers.actions.SuccessfulAuthAction
 import models.tcsp._
-import org.mockito.Matchers.any
+import org.mockito.Matchers.{any, eq => eqTo}
 import org.mockito.Mockito.{verify, when}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.mock.MockitoSugar
 import play.api.inject.bind
 import play.api.inject.guice.GuiceInjectorBuilder
-import play.api.test.Helpers._
+import play.api.test.Helpers.{BAD_REQUEST, OK, SEE_OTHER, contentAsString, redirectLocation, status, _}
```

Comment:
Can get rid of each specific and just leave _ in place of them.

Reply:
Refactored, cheers mate